galaxy_ng/app/api/v3/serializers/collection.py

```diff
@@ -90,6 +90,16 @@ class UnpaginatedCollectionVersionSerializer(_UnpaginatedCollectionVersionSerial
     class Meta(_UnpaginatedCollectionVersionSerializer.Meta):
         ref_name = "UnpaginatedCollectionVersionWithFixedHrefsSerializer"
 
+    def get_download_url(self, obj):
+        """
+        Get artifact download URL.
+        """
+
+        host = settings.ANSIBLE_API_HOSTNAME.strip("/")
+        prefix = settings.GALAXY_API_PATH_PREFIX.strip("/")
+        distro_base_path = self.context["path"]
+        return f"{host}/{prefix}/v3/artifacts/collections/{distro_base_path}/{obj.relative_path}"
```

Comment:
@gerrod3 what if we move this whole method https://github.com/ansible/galaxy_ng/blob/d1883acc636e7fa304211ca759e7aa3feeed4547/galaxy_ng/app/api/v3/serializers/collection.py#L104-L112

To the HrefNamespaceMixin https://github.com/ansible/galaxy_ng/blob/d1883acc636e7fa304211ca759e7aa3feeed4547/galaxy_ng/app/api/v3/serializers/collection.py#L24

?